CHANGES.txt

```diff
@@ -14,6 +14,7 @@ https://mhammond.github.io/pywin32_installers.html.
 Coming in build 309, as yet unreleased
 --------------------------------------
 
+* Updated scintilla constants in `Pythonwin/pywin/scintilla/scintillacon.py` (#2429, @Avasam)
```

Comment:
no need for this!

Reply:
Indeed since this will go in the same update as updating scintilla it's redundant